src/app/properties/property-new/property-new.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-property-new',
+  templateUrl: './property-new.component.html',
+  styleUrl: './property-new.component.css'
+})
+export class PropertyNewComponent {
```

Comment:
Is this component supposed to be outside of the `property-management` module?
If yes, then wouldn't it be better if `property-mangement` had a different name since it's just components for the form for adding a new property?